CPT331.WebAPI/Models/EventModel.cs

```diff
@@ -47,8 +48,8 @@ public EventModel(string address, DateTime beginDateTime, string description, Da
 			}
 		}
 
-		private const int BannerImageTransformationID = 7;
-		private const int ThumbnaiImageTransformationID = 15;
+		private const EventFinderImageFormat BannerImageTransformationID = EventFinderImageFormat.Size650x280;
+		private const EventFinderImageFormat ThumbnaiImageTransformationID = EventFinderImageFormat.Size75x75;
```

Comment:
I think it's a little more readable with an Enum... 

Although it seems I forgot to change to a more suitable variable name. 

Doh! 
/Uppercut
![IMG nuke](https://media1.giphy.com/media/HhTXt43pk1I1W/200_s.gif)